react/__tests__/ProductSummary.test.js

```diff
@@ -1,28 +1,61 @@
 import React from 'react'
-import { mount, shallow } from 'enzyme'
+import { render } from 'test-utils'
 
 import ProductSummary from '../index'
 
 describe('<ProductSummary /> component', () => {
   function renderComponent(customProps) {
-    const intl = {
-      formatMessage: {}
-    }
     const props = {
       ...customProps,
-      intl,
       runtime: { hints: {} },
+      product: {
+        linkText: 'linkText',
+        productName: 'productName',
+        sku: {
+          name: 'name',
+          itemId: 'itemId',
+          image: {
+            imageUrl: '',
+          },
+          seller: {
+            sellerId: 'sellerId',
+            commertialOffer: {
+              Installments: [
+                {
+                  Value: 1,
+                  InterestRate: 1,
+                  NumberOfInstallments: 1,
+                },
+              ],
+              Price: 1,
+              ListPrice: 1,
+            },
+          },
+        },
+        productClusters: [
+          {
+            name: 'name',
+          },
+        ],
+        quantity: 1,
+      },
     }
 
-    const component = shallow(<ProductSummary {...props} />)
-
-    return {
-      component,
-    }
+    return render(<ProductSummary {...props} />)
   }
 
-  it('should be mounted and not break', () => {
-    const { component } = renderComponent()
-    expect(component).toBeTruthy()
+  it('should be match the snapshot for normal mode', () => {
+    const { asFragment } = renderComponent({ displayMode: 'normal' })
+    expect(asFragment()).toMatchSnapshot()
```

Comment:
Every time someone uses react-testing-library a small little kitten is born.

Every time someone uses react-testing-library but only uses snapshot, the same kitten is sliced into pieces.


Save the kitten!

Reply:
(Use test matchers instead)

Reply:
We are only doing snapshot tests for now.
But don't worry, The next step is to make more complex tests

PS: Save the kittens